po/zh-CN.po

```diff
@@ -14459,14 +14461,17 @@ msgid ""
 "you\n"
 "can rely on the compiler to ensure correctness at runtime."
 msgstr ""
+"Rust 类型系统在使许多并发 bug 成为编译时 bug 方面\n"
```

Comment:
Just in case I haven't said it already: many of the English sentences are hard to translate directly and should probably be rephrased. You're all invited to send PRs to make them easier to understand.